src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/move/MovePersisterTest.kt

```diff
@@ -0,0 +1,121 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.move
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager
+import org.springframework.context.annotation.Import
+
+import org.springframework.test.context.ActiveProfiles
+import uk.gov.justice.digital.hmpps.pecs.jpc.TestConfig
+import uk.gov.justice.digital.hmpps.pecs.jpc.import.report.EventType
+import uk.gov.justice.digital.hmpps.pecs.jpc.import.report.FilterParams
+import uk.gov.justice.digital.hmpps.pecs.jpc.import.report.ReportJourneyWithEvents
+import uk.gov.justice.digital.hmpps.pecs.jpc.import.report.Report
+import uk.gov.justice.digital.hmpps.pecs.jpc.location.LocationRepository
+import uk.gov.justice.digital.hmpps.pecs.jpc.price.Price
+import uk.gov.justice.digital.hmpps.pecs.jpc.price.PriceRepository
+import uk.gov.justice.digital.hmpps.pecs.jpc.price.Supplier
+import uk.gov.justice.digital.hmpps.pecs.jpc.report.*
+import java.time.LocalDate
+import java.util.UUID
+
+@ActiveProfiles("test")
+@DataJpaTest
+@Import(TestConfig::class)
+internal class MovePersisterTest {
+
+    @Autowired
+    lateinit var locationRepository: LocationRepository
+
+    @Autowired
+    lateinit var priceRepository: PriceRepository
+
+    @Autowired
+    lateinit var moveRepository: MoveRepository
+
+    @Autowired
+    lateinit var moveQueryRepository: MoveQueryRepository
+
+    @Autowired
+    lateinit var journeyRepository: JourneyRepository
+
+    @Autowired
+    lateinit var entityManager: TestEntityManager
+
+    val from = LocalDate.of(2020, 9, 1)
+    val to = LocalDate.of(2020, 9, 6)
+
+
+    @Test
+    fun `Persist one move with one extant location and one journey`() {
```

Comment:
Could perhaps do with breaking this up into some smaller functions to aid test readability.....